composer.json

```diff
@@ -345,13 +347,18 @@
             "Generated\\": "src/Generated/",
             "Orm\\": "src/Orm/",
             "Pyz\\": "src/Pyz/",
-            "SprykerConfig\\": "src/SprykerConfig/"
+            "SprykerConfig\\": "src/SprykerConfig/",
+            "SprykerTest\\Shared\\FileManager\\Helper\\": "vendor/spryker/file-manager/tests/SprykerTest/Shared/FileManager/_support/Helper/",
```

Comment:
Will be removed after release of https://release.spryker.com/release-groups/view/5790